specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7260,6 +7260,10 @@
           "type": "boolean",
           "description": "Indicates whether the extension should use a newer minor version if one is available at deployment time. Once deployed, however, the extension will not upgrade minor versions unless redeployed, even with this property set to true."
         },
+        "enableAutomaticUpgrade": {
```

Comment:
is there default value for this property?

Reply:
Hi @yungezz the default value is null for this property.